### ISSUE TITLE
Schematic?

### ISSUE THREAD

**Issue by Shamr0x** [LukasGlader/Airsoft-Domination-Point#1]
Is it possible to post a schematic? I'm having issues assembling this.

**Comment by LukasGlader** [comment]
I added a circuit diagram now as well as updating the comments in the code.
The diagram is for an Arduino Uno. If you use another model check the pinout diagram for it and identify the SCL and SDA pins (A4 and A5 on the Uno) and connect accordingly.

**Comment by Shamr0x** [comment]
Is there a way to run this with a 1602A display?

**Comment by LukasGlader** [comment]
No. It needs to be a SH1106 or SSD1306 display using i2c, with a resolution of 128x64. I have used 0.96" and 1.3" OLED displays.

**Comment by Shamr0x** [comment]
Ok, I have a 1106 ordered. Also as for a board I have a elegoo uno r3 using a mega16u2. Would there be any compatability issues?

**Comment by LukasGlader** [comment]
I don't know for sure but I'm guessing it's an accurate clone so you should be good.
If you start substituting ingredients in a recipe you may have to learn how to cook. ;)
